t/ui/01-list.t

```diff
@@ -188,6 +201,27 @@ subtest 'available comments shown' => sub {
     );
     is(@{$driver->find_elements('#job_99962 .fa-comment')},
         0, 'available comments only shown if at least one comment available');
+    is(
+        $driver->find_element('#job_99936 .fa-bolt')->get_attribute('title'),
+        "Bug referenced: poo#1\nopen poo bug",
+        'available bugref (poo#1) shown for finished jobs'
+    );
+    is(
+        $driver->find_element('#job_99936 .fa-bug')->get_attribute('title'),
+        "Bug referenced: bsc#3",
+        'available bugref (bsc#3) shown for finished jobs'
+    );
+    my @closed = $driver->find_elements('#job_99936 .bug_closed');
+    is(
+        $closed[1]->get_attribute('title'),
+        "Bug referenced: poo#2\nclosed poo bug",
+        'available bugref (poo#2) shown for finished jobs'
+    );
+    is(
+        $closed[0]->get_attribute('title'),
+        "Bug referenced: bsc#4\nclosed bugzilla bug",
+        'available bugref (bsc#4) shown for finished jobs'
+    );
```

Comment:
you don't seem to have added an additional test for the case that caused the regression. Can you add something here or is it handled elsewhere?

Reply:
It's handled by `bsc#3` (has no corresponding bug)

Reply:
I added a comment to clarify that